### PR TITLE
Export schema summary

### PR DESCRIPTION
**Pull request by SteRiccio**
## Description

- #### Added/Removed/Updated something...
   
- #### Added/Removed/Updated something else...
    

## Types of changes

- [ ] Docs change
- [ ] Refactoring
- [ ] Dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Do you consider this PR needs further testing?

- [ ] No
- [ ] Yes

### CODE REVIEW

**Comment by minotogna** [comment]
Task linked: [CU-g13nh9 Export schema summary](https://app.clickup.com/t/g13nh9)

File: test/e2e/tests/surveySchemaSummary.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+  // sort items by path
+  return items.sort((item1, item2) => {
+    const path1 = item1.path
+    const path2 = item2.path
+    if (path1 > path2) return 1
+    if (path2 > path1) return -1
+    return 0
+  })
+}
```
Comment (ramirobg94) [inline test/e2e/tests/surveySchemaSummary.js:38]: maybe we can extract this because it is repeated

Comment (ramirobg94) [reply test/e2e/tests/surveySchemaSummary.js:38]: but this is a tests! okay!

**Review by ramirobg94 — APPROVED** [review]

File: test/e2e/tests/surveySchemaSummary.js (hunk 1 above)
Comment (SteRiccio) [reply test/e2e/tests/surveySchemaSummary.js:38]: uhm, right, where can we put it? it's something that can go to some ArrayUtils module, something like sortByProperty

**Review by ramirobg94 — APPROVED** [review]

**Pull request merged**